erigon-lib/state/aggregator_bench_test.go

```diff
@@ -287,3 +287,15 @@ func Benchmark_Recsplit_Find_ExternalFile(b *testing.B) {
 		require.EqualValues(b, keys[p], key)
 	}
 }
+
+func BenchmarkAggregator_BeginFilesRo(b *testing.B) {
+	//BenchmarkAggregator_BeginFilesRo/begin_files_ro-16  1737404  737.3 ns/op  3216 B/op  21 allocs/op
+	aggStep := uint64(100_00)
+	_, agg := testDbAndAggregatorBench(b, aggStep)
+
+	b.Run("begin_files_ro", func(b *testing.B) {
+		for i := 0; i < b.N; i++ {
+			agg.BeginFilesRo()
```

Comment:
i don't know: should we include rollback or not. rollback perf is important. but APP usually will be in state when "already exists many opened RoTx and we trying to open one more).

Reply:
temporal.BeginRoTx() + Rollback()
vs
agg.BeginFilesTx() + Close()

have similar throughputs (above 1-2 million goroutines, severe deterioration)